source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -1,167 +1,194 @@
+   to create and add new bundles to your mix with no extra migration step.
 
-.. code-block:: console
+#. **Install custom bundles**
 
-   $ sudo swupd update --migrate
+   Install your custom bundle using the normal swupd :command:`bundle-add`
+   command.
 
-After you migrate, the version of your |CL| system switches over to your
-last custom version number as noted in the previous section. 
+   View the `swupd man page`_ for more information on swupd commands.
 
-You can continue to create new bundles with :command:`mixin` 
-while you are in your custom version of |CL|.  You do not need to migrate
-again. However, you must run :command:`swupd update` again to update your
-system in order to make those bundles visible. 
+Examples
+********
 
-Add your custom bundle to your system
-*************************************
+The following examples use:
 
-#. Get a listing of your newly-created bundle.
+* A stock installation of |CL| with all `Prerequisites`_.
 
-   .. code-block:: console
+Example 1: Add custom helloclear bundle
+=======================================
+
+This example shows the basic steps of adding a custom bundle from a local repo.
 
-      $ sudo swupd bundle-list -a
+#. Check that `helloclear` does not exist on your system:
 
-   The listing includes all upstream bundles.
+   .. code-block:: bash
 
-#. Add your bundle.
+      helloclear
 
    .. code-block:: console
 
-      $ sudo swupd bundle-add [bundle-name]
+      helloclear: command not found
 
-.. note:: 
+#. Follow "Option 1: Build a new RPM" from :ref:`autospec` to create a new
+   `helloclear` RPM.
 
-   You can also update your system to the latest upstream version using
-   this command:   
-   
-   .. code-block:: console
+   The resulting RPMs are in `~/clearlinux/packages/helloclear/rpms`.
+
+#. Create a new repo.
+
+   #. Create a local repo folder and copy the new `helloclear` RPM files into
+   the repo:
+
+      .. code-block:: bash
+
+         mkdir ~/mixin-repo
+         cp ~/clearlinux/packages/helloclear/rpms/helloclear-v1.0-1.x86_64.rpm ~/mixin-repo
+         cp ~/clearlinux/packages/helloclear/rpms/helloclear-bin-v1.0-1.x86_64.rpm ~/mixin-repo
+
+   #. Create the repo data:
+
+      .. code-block:: bash
+
+         cd ~/mixin-repo
+         createrepo_c .
+
+   #. Add the repo name, making sure to use your username for the correct path:
+
+      .. code-block:: bash
 
-      $ sudo swupd update
+         sudo mixin repo add mylocalrepo file:///home/<usrname>/mixin-repo/
 
-Optional: Revert your system back to 100% upstream
-**************************************************
+#. Create custom bundle with the new `helloclear` RPM. Add `helloclear` to the
+   `helloclear-bundle` bundle and build the bundle:
+
+      .. code-block:: bash
+
+         sudo mixin package add helloclear --bundle helloclear-bundle
+         sudo mixin build
+
+#. Migrate your |CL| to your custom mix. Check your version before and after the
```

Comment:
Should this be "Check your version, before and after the update, to see the switch to your custom mix:"?